pytorch_lightning/metrics/functional/classification.py

```diff
@@ -332,107 +334,6 @@ def recall(
                             num_classes=num_classes, class_reduction=class_reduction)[1]
 
 
-def _binary_clf_curve(
```

Comment:
Can you share where you find the other version of the function ?

Reply:
`from pytorch_lightning.metrics.functional.precision_recall_curve import _binary_clf_curve`